Makefile

```diff
@@ -90,7 +90,7 @@ bin/gradualizer: $(beams) ebin/gradualizer.app
 
 .PHONY: gradualize
 gradualize: escript
-	bin/gradualizer --infer --solve_constraints --specs_override_dir priv/extra_specs/ \
+	@bin/gradualizer --infer --solve_constraints --specs_override_dir priv/extra_specs/ \
```

Comment:
You don't want the user to see which command we're running?

Reply:
It's connected with how we count lines in the CI self-check assessment (see changes to `.github/workflows/self-check.yml` in this PR). We cannot rely on the status code, because we ignore some trivial errors with `grep -v` and the status code is failing. We check the number of error lines instead. If we print the command, we have to assume exactly 2 lines means "no errors", which is a bit strange. If we do not print the command, 0 error lines means success.

All in all, it's a detail. The user can still see the command in the Makefile.